README.md

```diff
@@ -41,6 +41,18 @@ Finally, be sure to include the autoloader:
 require_once '/path/to/your-project/vendor/autoload.php';
 ```
 
+This library relies on `google/apiclient-services`. That library provides up-to-date API wrappers for a large number of Google APIs. In order that users may make use of the latest API clients, this library does not pin to a specific version of `google/apiclient-services`. **In order to prevent the accidental installation of API wrappers with breaking changes**, it is highly recommended that you pin to the latest version yourself prior to using this library in production.
+
+If you're installing this library for authentication or other purposes which do not require the API wrappers, you may prevent their installation by adding the following to your project's `composer.json` file:
```

Comment:
This is confusing to me, as it does not actually prevent their installation. Also, what is the use case for preventing their installation?

Reply:
Sorry, should have been `replace` not `provide`. There are a lot of examples of people using this library for convenience in authenticating HTTP clients (e.g. #1818, #1701) without actually using the services.